enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/types.ts

```diff
@@ -56,7 +56,7 @@ type InteractiveQuestionConfig = {
    * A callback function that triggers when a user saves the question. Only relevant when `isSaveEnabled = true`
    */
   onSave?: (
-    question: MetabaseQuestion | undefined,
+    question: MetabaseQuestion,
```

Comment:
I don't know why we say we would pass `undefined`. Like it doesn't make sense at all to pass `undefined`.

And we don't do that. We always pass a question
https://github.com/metabase/metabase/blob/051b61bd256c2337880f1354e69cb6a4dca2dbbc/enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx#L62

https://github.com/metabase/metabase/blob/051b61bd256c2337880f1354e69cb6a4dca2dbbc/enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx#L75